### DISCUSSION TITLE
Installed Apache Camel Karavan looks completely different to the one shown in tutorial videos and blog post

### DISCUSSION THREAD

**Discussion by TheHandOfNOD** [apache/camel-karavan#1019] [Q&A]
Hi,

I installed Apache Camel Karavan in 2 ways:

1. VS Code Plugin
2. Standalone in my Kubernetes cluster

Both installations look like the one in the screenshots:

![image](https://github.com/apache/camel-karavan/assets/10324233/788fdeec-9d81-4cf3-898f-eaa739bae357)

![image](https://github.com/apache/camel-karavan/assets/10324233/bda2ac2b-e4e7-46b8-bb10-0b960815e6c8)

In the Blog-Post and also in the videos Camel is shown like this, and contains lots more functionality as I am able to access currently. See for example the blog post of:
https://camel.apache.org/blog/2022/02/camel-karavan-0.0.12/

Here are some keyfacts about the installations I made:

VS Code Plugin:
- Karavan 4.1.0 Plugin
![image](https://github.com/apache/camel-karavan/assets/10324233/3b398b70-1145-47cf-9c9c-438c78af023e)

VS Code Version:
Version: 1.84.0 (Universal)
Commit: d037ac076cee195194f93ce6fe2bdfe2969cc82d
Date: 2023-11-01T11:30:19.406Z
Electron: 25.9.2
ElectronBuildId: 24603566
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.1.0

Kubernetes Web App (dev mode):
Image:          ghcr.io/apache/camel-karavan:4.1.0
Kubernetes: 1.28.0

I cannot figure out how it is possible to get the full application running, and to be honest there is not really useful documentation around Apache Camel Karavan at all, so I need to ask here for help how to get the real thing up and running.

Can you please provide some details? 
Am I missing something crucial?

I used the following instructions for the installation: 
https://github.com/apache/camel-karavan/blob/main/docs/WEB_KUBERNETES.md

Unfortunately also on the Camel Website: https://camel.apache.org/docs/
There is no link to the documentation for Apache Camel Karavan at all

![image](https://github.com/apache/camel-karavan/assets/10324233/c2c36918-527c-4488-8826-089ae5df6705)

Please help I am a little bit desperate right now (after searching documentation for a long time)

**Comment by TheHandOfNOD** [comment]
To be clear: I want my Karavan installation to look like this:

![image](https://github.com/apache/camel-karavan/assets/10324233/0401b98b-47e5-4b71-b607-be1995b96f41)

**Comment by davsclaus** [comment]
That link is to a blog post that is about 2 years old. Karavan keeps evolving and its UI gets more polished over time.

You can see an online editor at
https://karavan.space/

Karavan is primary here on github where you can find its docs, code, discussion etc.
https://github.com/apache/camel-karavan

**Comment by davsclaus** [comment]
And you can find more karavan blog posts at
https://camel.apache.org/categories/Karavan/

And there is also videos on you tube
https://www.youtube.com/@camel-karavan
